cfme/tests/infrastructure/test_tenant_quota.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.provisioning import do_vm_provisioning
+
+pytestmark = [
+    test_requirements.quota,
+    pytest.mark.meta(server_roles="+automate"),
+    pytest.mark.usefixtures('uses_infra_providers'),
+    pytest.mark.provider([VMwareProvider, RHEVMProvider], scope="module")
+]
+
+
+@pytest.fixture(scope="function")
+def vm_name():
+    vm_name = 'test_quota_prov_{}'.format(fauxfactory.gen_alphanumeric())
+    return vm_name
+
+
+@pytest.fixture(scope="function")
+def template_name(provisioning):
+    return provisioning["template"]
+
+
+@pytest.fixture(scope="module")
+def roottenant(appliance):
+    return appliance.collections.tenants.get_root_tenant()
+
+
+@pytest.fixture(scope="function")
+def prov_data(provider, provisioning):
+    return {
+        "first_name": fauxfactory.gen_alphanumeric(),
+        "last_name": fauxfactory.gen_alphanumeric(),
+        "email": "{}@{}.test".format(
+            fauxfactory.gen_alphanumeric(), fauxfactory.gen_alphanumeric()),
+        "manager_name": "{} {}".format(
+            fauxfactory.gen_alphanumeric(), fauxfactory.gen_alphanumeric()),
+        "vlan": provisioning.get("vlan"),
+        "datastore_name": {"name": provisioning["datastore"]},
+        "host_name": {"name": provisioning["host"]},
+        "provision_type": "Native Clone" if provider.type == "rhevm" else "VMware",
+        "catalog": {'vm_name': ''},
+        "environment": {'automatic_placement': True},
+    }
+
+
+@pytest.fixture(scope="function")
+def provisioner(provider):
```

Comment:
**Required/Question**

I don't see the advantage of this being a fixture, did you find it is used in other test modules? If not I'd request just calling the `do_vm_provisioning` function directly in the test instead of through a fixture.

Reply:
Changed it to just call the function `do_vm_provisioning` directly.